helpdesk_mgmt/models/helpdesk_ticket.py

```diff
@@ -132,6 +132,28 @@ def _read_group_stage_ids(self, stages, domain, order):
     )
     active = fields.Boolean(default=True)
 
+    def _send_mail_to_partner(self, template_name):
+        """Send a feedback message to the contact linked to this newly created ticket
+        (whatever it is a real partner or just its e-mail).
+        Note: The purpose of this function is to replace the stage's `mail_template_id`
+        role (that just doesn't work when the ticket is created from the portal or when
+        `partner_id` is set)"""
```

Comment:
**issue**: as far as I can tell, so far this doesn't replace but rather adds a new email, no? (see also the other comment below)